server/apps/prepopulate/data_initialization/validators.json

```diff
@@ -41,10 +41,16 @@
         },
         "subject" : {
             "type" : "list",
-            "required" : true
+            "required" : true,
+            "minlength" : 1
```

Comment:
Yep, with just the required : true it allows empty lists, need to minlength to ensure that there is at least one entry